automatic/sysinternals/tools/helpers.ps1

```diff
@@ -11,7 +12,7 @@ $tools = `
 "PsInfo",           "PsKill",                    "PsList",     "PsLoggedon",      "PsLoglist",
 "PsPasswd",         "PsService",                 "PsShutdown", "PsSuspend",       "RamMap",
 "RegDelNull",       "Regjump",                   "Regsize",    "RootkitRevealer", "Share Enum",
-"ShellRunas",       "SigCheck",                  "Streams",    "Strings",         "Sync",
+$ShellRunas,        "SigCheck",                  "Streams",    "Strings",         "Sync",
```

Comment:
Although not really necessary, it would still be better to quote the variable just to be sure.

```suggestion
"$ShellRunas",        "SigCheck",                  "Streams",    "Strings",         "Sync",
```